pulsar-io/jdbc/clickhouse/pom.xml

```diff
@@ -41,7 +41,6 @@
       <groupId>ru.yandex.clickhouse</groupId>
       <artifactId>clickhouse-jdbc</artifactId>
       <version>${clickhouse-jdbc.version}</version>
-      <scope>runtime</scope>
```

Comment:
Why remove the runtime scope here?

Reply:
need to user `ru.yandex.clickhouse.BalancedClickhouseDataSource`